test/system/research/user_logs_in_to_join_test.rb

```diff
@@ -31,7 +31,7 @@ class UserLogsInToJoinTest < TestCase
       visit new_user_session_path
       click_on "with GitHub"
 
-      assert_text "Current Research"
+      assert_selector 'h1', text: "Your research area"
```

Comment:
@kntsoriano I think if we're going to assert text, we should assert strong headings (h1/h2) like this. The risk is that by adding the text "current research" to the original page (which we could easily do), we silently destroy the meaning of this test and `click_on "with GitHub"` might fail but we wouldn't know.

Reply:
Yes. Agree with this approach.